tests/test_events.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from random import choice
+from uuid import uuid4
+
+from scripts.cat.cats import create_cat, Cat
+from scripts.cat.enums import CatRank
+from scripts.cat.sprites import sprites
+from scripts.clan import Clan
+from scripts.clan_package.settings import switch_clan_setting, set_clan_setting
+from scripts.events import events_class
+from scripts.game_structure import game
+from scripts.utility import get_living_cat_count
+
+
+class TestEvents(unittest.TestCase):
+    def test_bulk_skip(self):
+        # load in the spritesheets
+        # we have to do this to prevent a crash, even though we won't be displaying anything
+        sprites.load_all()
+
+        game.clan = Clan(
+            name=f"{'Test'}_{uuid4()}",
+            displayname="Test",
+            leader=create_cat(CatRank.LEADER),
+            deputy=create_cat(CatRank.DEPUTY),
+            medicine_cat=create_cat(CatRank.MEDICINE_CAT),
+            biome="Forest",
+            camp_bg="camp1",
+            symbol="ADDER0",
+            game_mode="expanded",
+            starting_members=[
+                create_cat(
+                    choice(
+                        [
+                            CatRank.KITTEN,
+                            CatRank.APPRENTICE,
+                            CatRank.WARRIOR,
+                            CatRank.WARRIOR,
+                            CatRank.ELDER,
+                        ]
+                    )
+                )
+                for _ in range(10)
```

Comment:
```suggestion
                for _ in range(7)
```
To better match the amount of cats when making a new Clan.

Reply:
Is that necessary though? I honestly just went with a round number for the heck of it, as well as figured that a larger number wouldn't hurt for suitability of the Clan.  idk that we *need* to replicate a player's starting Clan for this.

Reply:
That's fair.